sync/downstream.py

```diff
@@ -1057,6 +1103,7 @@ def update_pr(git_gecko, git_wpt, sync, action, merge_sha, base_sha, merged_by=N
         elif action == "closed":
             # We are storing the wpt base as a reference
             sync.data["wpt-base"] = base_sha
+            sync.get_gh_metadata()
```

Comment:
I"m not sure why it's called here?

Reply:
I think for the time being this makes sense here whilst we are trialing this, in the future it would make more sense for it to be in some refactored `try_notify` that also handles the notification of upstream CI tasks. 

This does result in a space leak currently, since we do not process the logs and remove them.